redux/meditationSlice.ts

```diff
@@ -27,6 +27,24 @@ const meditationSlice = createSlice({
         duration: action.payload,
       }
     },
+    manualEntry(
+      state,
+      action: PayloadAction<{
+        timestamp: number
+        duration: number
+      }>
+    ) {
+      const { duration, timestamp } = action.payload
+
+      if (duration === 0) {
+        delete state.activity[timestamp]
+        return
+      }
```

Comment:
If the duration is set to 0 it deletes the activity?

Reply:
Yeah, since there's no list to edit or delete afterwards I made the manual entry a single editable activity

Reply:
We might want to make that clear in the UI but all good :)